core/serv/src/main.rs

```diff
@@ -559,13 +559,9 @@ impl ServiceCommand {
                         .wrap(auth::Auth::default())
                         .route("/me", web::get().to(me))
                         .service(forward_gsb);
-                    if is_gsb_configured {
```

Comment:
I reverted everything to more or less original implementation (creating ClientProxy in [endpoint handler](https://github.com/golemfactory/yagna/pull/2056/commits/73ccff10c4fcd54d12fa384f09a7b02a59c2d14e#diff-3e6a65e63556a2ea137ec68bfeda8f092ae1351655220cf5e8a690ca1fcfa1a7R13)) because race condition seems to be caused by thread spawned in [Net::gsb](https://github.com/golemfactory/yagna/blob/master/core/net/src/hybrid/service.rs#L80), and I did not wanted to return some JoinHandle or sth to `ServiceCommand::run_command` because it is too long already.